sdk/selfhelp/Azure.ResourceManager.SelfHelp/src/Generated/Models/SearchResult.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.SelfHelp.Models
+{
+    /// <summary> Details of an AzureKB search result. </summary>
+    public partial class SearchResult
+    {
+        /// <summary> Initializes a new instance of SearchResult. </summary>
+        public SearchResult()
+        {
+        }
+
+        /// <summary> Initializes a new instance of SearchResult. </summary>
+        /// <param name="solutionId"> Unique id of the result. </param>
+        /// <param name="content"> Content of the search result. </param>
+        /// <param name="title"> Title of the search result. </param>
+        /// <param name="confidence"> Confidence of the search result. </param>
+        /// <param name="source"> Source of the search result. </param>
+        /// <param name="resultType"> Result type of the search result. </param>
+        /// <param name="rank"> rank of the search result. </param>
+        /// <param name="link"> Link to the document. </param>
+        internal SearchResult(string solutionId, string content, string title, Confidence? confidence, string source, ResultType? resultType, int? rank, string link)
+        {
+            SolutionId = solutionId;
+            Content = content;
+            Title = title;
+            Confidence = confidence;
+            Source = source;
+            ResultType = resultType;
+            Rank = rank;
+            Link = link;
+        }
+
+        /// <summary> Unique id of the result. </summary>
+        public string SolutionId { get; set; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/maps/Azure.ResourceManager.Maps/src/autorest.md?plain=1#L25).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/search/Azure.Search.Documents/src/autorest.md?plain=1#L106-L115).

Reply:
As earlier mentioned, this is string